storybook/storybook-react/src/components/OrderedList/OrderedList.docs.mdx

```diff
@@ -1,15 +1,52 @@
-import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import { Canvas, Meta } from "@storybook/blocks";
 import * as OrderedListStories from "./OrderedList.stories.tsx";
-import README from "../../../../../packages/css/src/ordered-list/README.md?raw";
 
 <Meta of={OrderedListStories} />
 
-<Markdown>{README}</Markdown>
+# Ordered list
```

Comment:
There are / were two reasons for importing the readme here:

1. If you have the same documentation in multiple places, it's easier / more maintainable to define it once and use it multiple times. We currently don't have multiple storybooks anymore, but in the future we will, so this might become relevant again.
2. If you look up the component directly on github, you would have (part of) the docs there as well. 

I agree that we should rethink this approach, but maybe we should make a decision on that first, before implementing a different approach in this PR?

Reply:
Yes, very valid points.

I’ve reinstated a minimal `README.md` and added a high priority Story to the backlog to improve the documentation approach.